src/Persistence/MassTransit.MongoDbIntegration/Configuration/MessageDataRepositorySelectorExtensions.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace MassTransit.MongoDbIntegration.Configuration
+{
+    using System;
+    using MassTransit.MessageData;
+    using MessageData;
+
+
+    public static class MessageDataRepositorySelectorExtensions
+    {
+        public static IMessageDataRepository UsingMongoDbStorage(this IMessageDataRepositorySelector selector, string connectionString, string containerName = default)
```

Comment:
Oops. I ended up going with `With` naming pattern and forgot this one - I'm not sure if you like that naming pattern anyway so I'll fix once I have feedback from you.